src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -140,6 +145,7 @@ const ContextMenu = createClass({
 			}
 		}, 10);
 
+		document.body.addEventListener('touchstart', this.handleBodyClick);
```

Comment:
should body click be triggered on `touchstart` or `touchend`?  I think `touchend` by default triggers `click` on most other DOM elements by default.